### PR TITLE
Update license-normalizer-bundle.json

### PR DESCRIPTION
**Pull request by ischerbinin**
Added some patterns for licences from model's virtual env

### CODE REVIEW

**Review by Andrei-Shulika — APPROVED** [review]

File: license-compliance/gradle-license-report/license-normalizer-bundle.json [hunk 1]
```diff
@@ -201,6 +224,10 @@
       "bundleName": "apache2",
       "licenseNamePattern": "^Apache License,? 2\\.0$"
     },
+    {
+      "bundleName": "apache2",
+      "licenseNamePattern": "^Apache Software License$"
```
Comment (OptimumCode) [inline license-compliance/gradle-license-report/license-normalizer-bundle.json:229]: question: Are we sure that this is Apache 2.0? The license name looks more like an older version of Apache. Can we check license text?

File: license-compliance/gradle-license-report/license-normalizer-bundle.json [hunk 2]
```diff
@@ -245,6 +280,10 @@
       "bundleName": "bsd3clause",
       "licenseNamePattern": "^The 3-Clause BSD License$"
     },
+    {
+      "bundleName": "bsd3clause",
+      "licenseNamePattern": "^BSD License$"
```
Comment (OptimumCode) [inline license-compliance/gradle-license-report/license-normalizer-bundle.json:285]: issue: This is a different BSD license. Probably this one: https://spdx.org/licenses/BSD-1-Clause.html. Can we check the license text?

File: license-compliance/gradle-license-report/license-normalizer-bundle.json [hunk 3]
```diff
@@ -453,10 +508,18 @@
       "bundleName": "apache2-mpl11",
       "licenseNamePattern": "^\\(Apache-2\\.0 OR MPL-1\\.1\\)$"
     },
+    {
+      "bundleName": "apache2-bsd3",
+      "licenseNamePattern": "^Apache Software License\\; BSD License$"
```
Comment (OptimumCode) [inline license-compliance/gradle-license-report/license-normalizer-bundle.json:513]: note: We don't need to escape `;` in regex (as far as I know)

File: license-compliance/gradle-license-report/license-normalizer-bundle.json (hunk 1 above)
Comment (ischerbinin) [reply license-compliance/gradle-license-report/license-normalizer-bundle.json:229]: "cyclonedx-python-lib","7.5.1","Apache Software License" - output in pip-licenses

https://github.com/CycloneDX/cyclonedx-python-lib/blob/main/LICENSE - license itself, in title Apache Version 2.0

File: license-compliance/gradle-license-report/license-normalizer-bundle.json (hunk 2 above)
Comment (ischerbinin) [reply license-compliance/gradle-license-report/license-normalizer-bundle.json:285]: Same here 
"flatdict","4.0.1","BSD License" - in pip-licenses report
License: BSD License (BSD 3-Clause License) - pypi.org

https://github.com/gmr/flatdict/blob/master/LICENSE - license itself

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**